src/index.js

```diff
@@ -1,12 +1,28 @@
-import _ from 'lodash';
+import Task from './modules/Task.js';
+import { list } from './modules/elements.js';
 import './style.css';
+import listItem from './modules/list-item';
 
-function component() {
-  const element = document.createElement('div');
+const init = () => {
+  const task1 = new Task('wash the dishes');
+  const task2 = new Task('complete to Do list project');
+  task1.appendToList();
+  task2.appendToList();
+  list.innerHTML += Task.list();
```

Comment:
- [ ] You are to write a function that takes the tasks array as an argument and populate your HTML 👍 Kindly rectify this issue. Take note that you do not need to implement any extra functionality but kudos for currently implementing the add tasks functionality 🔥 
- Requirements:
![Screenshot 2022-07-07 090553](https://user-images.githubusercontent.com/85113567/177725615-b09f65b5-6871-46f3-9bf5-353fa08f055d.png)

Reply:
@Kingstalux  [Here is my function ](https://github.com/bienvenuushindi/to-do-list/blob/list-structure/src/modules/Task.js#L16-L18) 

I think I implemented everything as  requested  I just did it implicitly